bin/etas/update_overlap_table.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "pathname"
+$LOAD_PATH.unshift(Pathname.new(__dir__).parent + "../lib")
+
+require "bundler/setup"
+require "cluster"
+require "cluster_overlap"
+require "logger"
+require "services"
+require "utils/waypoint"
+require "date"
+require "etas/overlap_table_update"
+
+Services.mongo!
+
+TABLENAME = :holdings_htitem_htmember
+BATCH_SIZE = 100_000
+
+if __FILE__ == $PROGRAM_NAME
+
+  # Default: Now - 36 hours
+  cutoff_date = Date.today - 1.5
+
+  unless ARGV.empty?
+    cutoff_date = Date.parse(ARGV.shift)
+  end
+
+  ETAS::OverlapTableUpdate.new(cutoff_date).run
```

Comment:
I'm not sure this is really the right name. This table existed before ETAS and is used for other (albeit less visible and perhaps in some cases theoretical) things like section 108 access and direct SSD access (for Michigan anyway) and for driving 'held by my library' facets in solr.

Prior to ETAS, we generally referred to this table as the "production holdings table". I think calling the "overlap table" is more specific while still being accurate. I'm not immediately sure where to put it -- it has some affinity with reports but isn't a report as such. That may be an argument for leaving it where it is (i.e. not in a submodule) for the time being esp. as ultimately we want to see if we can do away with it and have an API direct to the source instead.

Reply:
Perhaps the right concept is something like "production holdings" or "relational database holdings", but nothing immediate comes to mind as far as a better choice for the module name here.

Reply:
I gave up on giving these a module name and moved back into lib.